nikola/conf.py.in

```diff
@@ -645,6 +645,21 @@ COMMENT_SYSTEM_ID = ${COMMENT_SYSTEM_ID}
 # Enable comments on picture gallery pages?
 # COMMENTS_IN_GALLERIES = False
 
+# Use a thumbnail (defined by ".. previewimage:") in list of galleries
+# for each gallery
+GALLERIES_USE_THUMBNAIL = True
+
+# Columns of thumbnails list in galleries index
+GALLERIES_COLUMNS = 3
```

Comment:
if you use THUMBNAIL_SIZE as a guide for the sice of the thumbs, then this is probably not needed, just put them one after the other and as many as can fil will fit.

Keeping this setting means we have to ask themes to respect it and that's going to be difficult.